test/python/qasm3/test_export.py

```diff
@@ -948,7 +948,7 @@ def test_old_alias_classical_registers_option(self):
 
     def test_simple_for_loop(self):
         """Test that a simple for loop outputs the expected result."""
-        parameter = Parameter("x")
+        parameter = Parameter("my_x")
```

Comment:
Is there a reason you changed the name here? I assume it's because of the conflict between the name `x` and the gate. I saw the code above handling this, but is this a behavior change we should account for in a release note?

Reply:
`x` is the name of a gate in `stdgates.inc` (same for `y` and `t`, also renamed), and the OQ3 exporter was falsely allowing the shadowing previously, which got fixed when I formalised the idea of variables being in the symbol table.  I should add an explicit test and a bugfix note for it for these parameters, yeah.

Reply:
Test and release note added in 5fbeabacba.